omero/export_scripts/Batch_ROI_Export.py

```diff
@@ -305,9 +305,9 @@ def run_script():
             description="List of Dataset IDs or Image IDs").ofType(rlong(0)),
 
         scripts.List(
-            "Channels", grouping="3", default=[1L, 2L, 3L, 4L],
+            "Channels", grouping="3", default=[1, 2, 3, 4],
             description="Indices of Channels to measure intensity."
-            ).ofType(rlong(0)),
```

Comment:
In general, `int` makes sense here, but it will impact clients, right?

Reply:
I will have to double check

Reply:
I would think that webclient should handle this fine since the UI for that script is not hard-coded.